.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+    "overrides": [
+      {
+        "files": ["*.html"],
```

Comment:
@MathyouMB are you not getting it to format JS as well? It does it pretty well imo

Reply:
It does format the js and css, but it uses the default config.

Reply:
The config I added is specifically for working with the go templating syntax.

Reply:
Ah I see